sormas-backend/src/test/java/de/symeda/sormas/backend/TestDatabaseCleaner.java

```diff
@@ -25,12 +26,12 @@ public class TestDatabaseCleaner implements DatabaseCleaner {
 
 	public void run(Connection connection) throws SQLException {
 
+		// needed to drop h2 functions
 		connection.prepareStatement("DROP ALL OBJECTS").execute();
 		connection.commit();
 
 		Map<String, String> properties = new HashMap<>();
-		properties.put("hibernate.hbm2ddl.auto", "create");
-		Persistence.generateSchema("beanTestPU", properties);
+		properties.put(AvailableSettings.HBM2DDL_DATABASE_ACTION, "create");//Action.CREATE.getExternalJpaName());
```

Comment:
Add marker `// TODO #11032 Switch to constant from org.hibernate.tool.schema.Action`